models.py

```diff
@@ -0,0 +1,36 @@
+import json
+
+
+class JSONModel:
+    def __init__(self, json_file="scratch.json"):
+        self.json_file = json_file
+        try:
+            with open(self.json_file, 'r') as data_file:
+                self.data = json.load(data_file)
+        except (FileNotFoundError, json.decoder.JSONDecodeError):
+            self.data = []
+
+    def add(self, record):
+        self.data.append(record)
+        with open(self.json_file, 'w') as data_file:
+            json.dump(self.data, data_file)
+
+
+class Task(JSONModel):
+    def __init__(self, **kwargs):
+        for k, v in kwargs.items():
+            setattr(self, k, v)
```

Comment:
In the future, you may want to do some validation and ensure the expected fields are present and add specific gettr methods to your class.